src/components/Permissions/DataList.jsx

```diff
@@ -1,15 +1,117 @@
+/* eslint-disable no-unsafe-optional-chaining */
```

Comment:
let's not disable this rule here please.

when we add this rule, we should add it close to the code, and explain why we disable it. I advise you to read the rule purpose inside Eslint documentation, and if we find a good reason not to respect it inside on line, then, disable the line and add a comment to explain why